torch/testing/_internal/common_fsdp.py

```diff
@@ -307,11 +312,28 @@ def init(
                 )
             else:
                 auto_wrap_policy = fsdp_kwargs.pop("auto_wrap_policy")
+
+            if (
+                "sharding_strategy" in fsdp_kwargs
+                and fsdp_kwargs["sharding_strategy"] in {
+                    ShardingStrategy.HYBRID_SHARD,
+                    ShardingStrategy.HYBRID_SHARD_ZERO2
```

Comment:
Missed one (which is causing the test failures):
```suggestion
                    ShardingStrategy._HYBRID_SHARD_ZERO2
```